### ISSUE TITLE
Left menu on mobile

### ISSUE THREAD

**Issue by tchiotludo** [kestra-io/kestra#1425]
Left menu on mobile should be forced reduced at least, or even more hidden by default

**Comment by MRADUL9936** [comment]
@anna-geller  can u provide the issue pic and any design  to update the ui, because i am not able to get the issue, i checked the website ui working fine in mobile phone too.

**Comment by anna-geller** [comment]
sure, a small explanation here https://share.descript.com/view/KryZRUNQquR

**Comment by MRADUL9936** [comment]
@anna-geller How about a hamburger menu? I'd be happy to implement it if you'd like.

**Comment by tchiotludo** [comment]
yes hamburger menu will be the best solution, hide the whole sidebar on mobile and add a hamburger menu, to display the expanded version of the menu only

**Comment by MRADUL9936** [comment]
@tchiotludo  then can you assign it to me.

**Comment by MilosPaunovic** [comment]
Hi @MRADUL9936,

Since there hasn't been any progress on the issue, I've unassigned you for now. If you decide to work on it and open a PR, let me know, and I'll be happy to reassign it to you.

In the meantime, the issue is open for anyone in the community who would like to take it on! 🚀

**Comment by MRADUL9936** [comment]
Hi @MilosPaunovic,
Could you provide some design suggestions for the sidebar menu? I've attached a reference design that i can implement If you like . 
https://github.com/user-attachments/assets/caed5cb7-7888-4a0d-9d2b-37ec6d8dc328

**Comment by anna-geller** [comment]
let me ping @Nico-Kestra for the design

**Comment by Nico-Kestra** [comment]
I'll look into it this week. I need to think about how to integrate it with the right menu as well

**Comment by anna-geller** [comment]
@Nico-Kestra how is this looking? if you can, let's validate this and add a figma link, might be a quick win in the end

**Comment by Nico-Kestra** [comment]
<img width="1259" alt="Image" src="https://github.com/user-attachments/assets/c765941d-7fc5-4053-8cb6-2a2f3e51a5b3" />

I don't know if you remember, but I had started working on this.

**Comment by Malavi1** [comment]
Hi @MilosPaunovic if the issue still open I would like to work on it. 
Can you get assign?!

**Comment by MilosPaunovic** [comment]
Hi @Malavi1,

Is there any progress on this?

**Comment by MilosPaunovic** [comment]
Hi @Malavi1,

As there hasn't been any progress or updates on this, I'll unassign you for now. However, if you do open a PR, it will be reviewed, and you'll be reassigned if no one else handles it in the meantime.

Thanks for all the efforts so far! 🍀

**Comment by PoorvaRaut** [comment]
Hii, if the issue still open I would like to work on it.
Can you assign?!

**Comment by Piyush-r-bhaskar** [comment]
Go for it @PoorvaRaut ..Thanks 🚀

**Comment by MilosPaunovic** [comment]
Hi @PoorvaRaut,

As there hasn't been any progress or updates on this, I'll unassign you for now. However, if you do open a PR, it will be reviewed, and you'll be reassigned if no one else handles it in the meantime.

Thanks for all the efforts so far! 🍀

**Comment by PoorvaRaut** [comment]
Heyy.. i m working on it... i will let you know uptill end of the day

**Comment by MilosPaunovic** [comment]
Hi @darshanCode2005, thank you so much for opening a PR for this, it's really appreciated! 🚀 

If you add a comment here, I'll be able to assign you to the issue; otherwise, I cannot due to GitHub's privacy policies.

**Comment by DarshanCode2005** [comment]
Hi @anna-geller, @MilosPaunovic , @tchiotludo ,
I hope you’re doing well. I’ve submitted a PR here: https://github.com/kestra-io/kestra/pull/12072. The solution is fairly straightforward, but I’d really appreciate it if you could take a look and share your feedback.

If any edits or improvements are needed, please do let me know. Thank you so much for your time and guidance!

**Comment by Piyush-r-bhaskar** [comment]
Thank you for this @DarshanCode2005 

Few comments in PR fo you to have a look 🚀

**Comment by DarshanCode2005** [comment]
I have implemented the changes you suggested, @Piyush-r-bhaskar, to add the hamburger menu with a closing icon for the mobile viewport.

**Comment by MilosPaunovic** [comment]
Unassigned due to https://github.com/kestra-io/kestra/pull/12072#issuecomment-3424956393.

**Comment by MilosPaunovic** [comment]
This is still open to external contributors as a `good first issue` in our repository.

**Comment by mahenoorsalat** [comment]
I think better than shrink we should overlay the nav cause if it's overlay then we can make it minimize and maxmize the way we want ..

should I handle this ?? @MilosPaunovic @Piyush-r-bhaskar

**Comment by MilosPaunovic** [comment]
This will be handled as part of https://github.com/kestra-io/kestra-ee/issues/5415.
